django/db/backends/postgresql/compiler.py

```diff
@@ -0,0 +1,41 @@
+from django.db.models.sql.compiler import (
+    SQLAggregateCompiler,
+    SQLCompiler,
+    SQLDeleteCompiler,
+    SQLInsertCompiler as BaseSQLInsertCompiler,
+    SQLUpdateCompiler,
+)
+
+
+class InsertUnnest(list):
```

Comment:
This is really just a sentinel object to respect the current signature of `SQLInsertCompiler.assemble_as_sql`.